p2p/simulations/http.go

```diff
@@ -320,7 +320,8 @@ func (s *Server) StartMocker(w http.ResponseWriter, req *http.Request) {
 	mockerid := req.Context().Value("mock").(string)
 
 	if len(s.Mockers) == 0 {
-		http.Error(w, "mocker not configured", http.StatusInternalServerError)
+		//don't require a mocker to be present
+		s.JSON(w, http.StatusNotModified, "No mocker configured")
```

Comment:
I actually think the previous behaviour was correct, if you try and start a mocker on a backend which doesn't have one, then you are doing something wrong?